test/db/cmd/cmd_graph

```diff
@@ -636,39 +636,39 @@ s sym.ignore_call
 agF
 EOF
 EXPECT=<<EOF
-.-------------.
-|  0x80000c3  |
-`-------------'
+.---------------.
+|  0x80000c3 e  |
```

Comment:
I wonder if it's better to put letters into braces e.g. `(e)` or `{e}`, so they will be more separate from the number, especially those that are in hex range, e.g. `e`.